frontend/src/pages/pipelines/global/modelCustomization/startRunModal/StartRunModal.tsx

```diff
@@ -40,85 +55,77 @@ const StartRunModal: React.FC<StartRunModalProps> = ({
           isSubmitDisabled={!canContinue || !loaded}
         />
       }
-      variant="medium"
+      variant={ModalVariant.medium}
       data-testid="start-run-modal"
```

Comment:
You may need to add this to `ModalBody` component to fix the tests because there is no body wrapper now. The previous data-testid would apply to `ModalBody`.